servers/quarkus-common/src/main/java/org/projectnessie/quarkus/providers/storage/PersistProvider.java

```diff
@@ -134,23 +129,8 @@ public Persist producePersist() {
 
     LOGGER.info("Creating/opening version store {} ...", versionStoreType);
 
-    BiFunction<Persist, String, Persist> wrapPersistTracing = (p, name) -> p;
-    String tracingInfo = "without tracing";
-    if (versionStoreConfig.isTracingEnabled()) {
-      if (opentelemetryTracer.isUnsatisfied()) {
-        LOGGER.warn(
-            "OpenTelemetry is enabled, but not available, forgot to add quarkus-opentelemetry?");
```

Comment:
It's a bit sad that we are losing the ability to warn users in such cases. But I guess we can live with that.